client/app/vps/dashboard/vps-dashboard.controller.js

```diff
@@ -272,10 +289,14 @@ export default class {
         callback: () => this.VpsActionService.reboot(this.serviceName),
         isAvailable: () => !this.loaders.polling && !this.vps.loading,
       },
+      rebuild: {
+        callback: () => this.$state.go('iaas.vps.detail.dashboard.rebuild', { serviceName: this.serviceName }),
+        isAvailable: () => !this.loaders.polling && !this.vps.loading && this.isUSVps,
+      },
       reinstall: {
         text: this.$translate.instant('vps_configuration_reinstall_title_button'),
         callback: () => this.VpsActionService.reinstall(this.serviceName),
-        isAvailable: () => !this.loaders.polling && !this.vps.loading,
+        isAvailable: () => !this.loaders.polling && !this.vps.loading && !this.isUSVps,
```

Comment:
Can't we get that from the capabilities instead of using `isUSVps`?

Reply:
Sure :)